doc/workflow/coding_standard_qml.md

```diff
@@ -0,0 +1,175 @@
+# Coding Standard QML
+
+###1. QML document structure
+
+#####Rule 1.1. Basic document and elements structure should be order like this:
```

Comment:
Grammar: "should be ordered" or you could even re-phrase whole setence: "Elements in the qml file should be ordered for each element in the following way:"